astropy/units/quantity.py

```diff
@@ -428,7 +427,7 @@ def __array_wrap__(self, obj, context=None):
 
         raise NotImplementedError('__array_wrap__ should not be used '
                                   'with a context any more, since we require '
-                                  'numpy >=1.16.  Please raise an issue on '
+                                  'numpy >=1.17.  Please raise an issue on '
```

Comment:
Let's remove the version here - maybe "since all use should go through __array_function__. Please ..."